### PR TITLE
Fix mistakes in UG

### PR DESCRIPTION
**Pull request by Ma-Yueran**
There are some mistakes because of the update of images.

### CODE REVIEW

**Comment by Ma-Yueran** [comment]
Update command summary

Separate examples from format

**Review by HynRidge — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -321,11 +321,15 @@ Displays help for all/certain commands.
...
+Showing all commands:
 ![help](images/screenshots/help.png)
 
+Showing one specific command:
+![help](images/screenshots/help2.png)
+
 Examples : 
 * `help` (Shows all command and its format)
-* `help tag` (shows tag command description, format and examples usage)
+* `help tag` (shows cd command description, format and examples usage)
```
Comment (luo-git) [inline docs/UserGuide.md]: Shouldn't this still be showing help for tag?

**Review by luo-git — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (Ma-Yueran) [reply docs/UserGuide.md]: The help2 image I added is cd, so I changed it to cd

Comment (Ma-Yueran) [reply docs/UserGuide.md]: emmm, just found did not change the help tag to help cd

Comment (Ma-Yueran) [reply docs/UserGuide.md]: thank you for pointing that out

**Review by luo-git — APPROVED** [review]

**Pull request merged**